test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -185,6 +185,13 @@ Describe "New-Item" -Tags "CI" {
             Pop-Location
         }
     }
+
+    It "Should display an error message when a symbolic link target is not specified" {
+        { New-Item -Name:$testfile -Path:$tmpDirectory -ItemType:SymbolicLink } | Should -Throw -ErrorId 'ArgumentNull,Microsoft.PowerShell.Commands.NewItemCommand'
+        $Error | Select-Object -First:1 | Foreach-Object Exception | Foreach-Object ParamName | Should -Be 'content'
+        { New-Item -Name:$testfile -Path:$tmpDirectory -ItemType:SymbolicLink -Value:{} } | Should -Throw -ErrorId 'ArgumentNull,Microsoft.PowerShell.Commands.NewItemCommand'
+        $Error | Select-Object -First:1 | Foreach-Object Exception | Foreach-Object ParamName | Should -Be 'content'
```

Comment:
Please follow common test style.
We can safely use `$Error[0]`.
We prefer to use `-BeExactly` for strings.
```suggestion
        { New-Item -Name $testfile -Path $tmpDirectory -ItemType SymbolicLink } | Should -Throw -ErrorId 'ArgumentNull,Microsoft.PowerShell.Commands.NewItemCommand'
        $Error[0].Exception.ParamName | Should -BeExactly 'Content'
        { New-Item -Name $testfile -Path $tmpDirectory -ItemType:SymbolicLink -Value {} } | Should -Throw -ErrorId 'ArgumentNull,Microsoft.PowerShell.Commands.NewItemCommand'
         $Error[0].Exception.ParamName | Should -BeExactly 'Content'
```